package.json

```diff
@@ -241,10 +242,7 @@
     "template": "COMMIT_TEMPLATE"
   },
   "pre-push": {
-    "run": [
-      "lint",
-      "pre-push-tests"
-    ]
```

Comment:
should this have been removed? (btw you can delete the commit hooks in `.git/hooks` instead so it doesn't change any files that are checked into git)

Reply:
ya shouldn't be removed. added back